test/js/search/more-facets.test.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import * as ReactAll from 'react';
+import { mount } from 'enzyme';
+import * as Reach from '@reach/router'
+
+const fetch = require('node-fetch');
+
+import MoreFacetsButton from 'components/MoreFacetsButton';
+import StudySearchProvider from 'components/search/StudySearchProvider';
+import * as ScpAPI from 'lib/scp-api'
```

Comment:
Imports like these, that rely on Rails `webpacker.yml`, prevent developers from using out-of-the-box IDE features like `Look up definition`, `Rename symbol`, and displaying docstrings on mouse-hover.

Relative imports enable such developer productivity features by default.  Specifically:
```suggestion
import MoreFacetsButton from '../../app/javascript/components/MoreFacetsButton';
import StudySearchProvider from '../../app/javascript/components/search/StudySearchProvider';
import * as ScpAPI from '../../app/javascript/lib/scp-api'
```

Reply:
per our chat, I'll keep these for now, and investigate ways to enable IDE discovery of them